src/components/layout/nav/PrivateMenu.tsx

```diff
@@ -73,7 +73,7 @@ export default function PrivateMenu() {
       <IconButton onClick={handleMenu} size="medium">
         {session?.user?.picture ? (
           <Avatar
-            sx={{ width: theme.spacing(4), height: theme.spacing(4) }}
+            sx={{ width: 200, height: 200 }}
```

Comment:
Isn't that a bit too much for an avatar? Is that width and height what is actually rendered?

Reply:
I had a problem where this file and the other were not updated so i was seeing the old dimensions. Fixed it